backend/src/memos/memos.controller.ts

```diff
@@ -51,4 +52,14 @@ export class MemosController {
   ): Promise<void> {
     await this.memosService.uploadMemosComplete()
   }
+
+  /**
+   * Get memo based on slug
+   */
+  @Get(':slug')
+  @ApiResponse({ type: GetMemoResponseDto })
+  async getMemo(@Param('slug') slug: string): Promise<GetMemoResponseDto> {
```

Comment:
No big deal, but if you just use `@Param('slug')` here, you don't need `GetMemoDto`